README.md

```diff
@@ -831,9 +865,13 @@ A new, deeply immutable, compound primitive type data structure, proposed in thi
 
 A new, deeply immutable, compound primitive type data structure, proposed in this document, that is analogous to Array. `#[1, 2, 3, 4]`
 
+#### Box
+
+A new primitive type that can store a reference to an object, proposed in this document. `Box({})`.
+
 #### Compound primitive types
 
-Values which act like other JavaScript primitives, but are composed of other constituent values. This document proposes the first two compound primitive types: `Record` and `Tuple`.
+Values which act like other JavaScript primitives, but are composed of other constituent values. This document proposes the first three compound primitive types: `Record`, `Tuple` and `Box`.
```

Comment:
describing `Box` as a compound primitive doesn't feel right, but I can't put my finger on exactly why. I also don't have a better suggest. So feel free to ignore me here!

Reply:
Yeah, it's not really "compound" because it contains a single value, but it still isn't an "atomic" primitive since it contains something.